libraries/render-utils/src/MeshPartPayload.cpp

```diff
@@ -23,292 +22,33 @@
 
 #include "RenderPipelines.h"
 
-// static const QString ENABLE_MATERIAL_PROCEDURAL_SHADERS_STRING { "HIFI_ENABLE_MATERIAL_PROCEDURAL_SHADERS" };
```

Comment:
this class was a bit of a mess and I've alway wanted to clean it up.  basically, it contained both the base class MeshPartPayload and a deriving class, ModelMeshPartPayload.  but we never actually instantiate a plain "MeshPartPayload," and as a result all of the MeshPartPayload was unmaintained.  in the process of adding support for billboarding, I've combined and simplified them